Piedone.HelpfulLibraries.csproj

```diff
@@ -12,14 +12,15 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>

```

Comment:
The changes in this file are based on the changes GH has made to their own copy of HL [here](https://bitbucket.org/global-health/piedone.helpfullibraries/src/master/Piedone.HelpfulLibraries.csproj):

![image](https://user-images.githubusercontent.com/164360/145193694-324db4ee-44fc-47e7-8d8e-45f29e1087a3.png)